src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/PublicAPI.Shipped.txt

```diff
@@ -105,7 +105,6 @@ Microsoft.VisualStudio.ProjectSystem.VS.Interop.IVsAppId.Initialize() -> int
 Microsoft.VisualStudio.ProjectSystem.VS.Interop.IVsAppId.SetGuidProperty(int propid, ref System.Guid rguid) -> int
 Microsoft.VisualStudio.ProjectSystem.VS.Interop.IVsAppId.SetProperty(int propid, object var) -> int
 Microsoft.VisualStudio.ProjectSystem.VS.Interop.IVsAppId.SetSite(Microsoft.VisualStudio.OLE.Interop.IServiceProvider pSP) -> int
-Microsoft.VisualStudio.ProjectSystem.VS.Interop.SVsAppId
```

Comment:
I don't know why these are public, but is it safe to assume nobody is depending on us?